pkg/scheduler/framework/plugins/volumebinding/volume_binding_test.go

```diff
@@ -890,3 +891,90 @@ func TestVolumeBinding(t *testing.T) {
 		})
 	}
 }
+
+func TestIsSchedulableAfterCSIStorageCapacityChange(t *testing.T) {
+	table := []struct {
+		name   string
+		pod    *v1.Pod
+		oldCap interface{}
+		newCap interface{}
+		err    bool
+		expect framework.QueueingHint
+	}{
+		{
```

Comment:
nit: It may be better to add a test case for pod with both pvcs and generic ephemeral volumes.